### PR TITLE
Updated to match ContentDB prepped by Awkanimus

### PR DESCRIPTION
**Pull request by Awkanimus**
- Content from ContentDB copied and overlayed on top of existing repo
 - No actual changes made by Awkanimus
 - Does assume that all content from ContentDB download should be in VCS

As I didn't make any changes, I'd prefer to do a --author=AtlanteFr but it felt weird to do outside of a professional setting without permission. If you're fine with it, your name will be attached to the commit and mine will just be the committer.

Of course, it could also make sense to just close this as you already have the updated source. Either way's fine with me.

### CODE REVIEW

**Comment by AtlanteFr** [comment]
Could you rephrase your post, I can't understand what you mean.

**Comment by Awkanimus** [comment]
As requested, I reformulated the description.

**Comment by Awkanimus** [comment]
Just in case it's specifically setting the author which is causing the confusion here's an explanation. Apologies if you already know all this:

You can do the following:
1. Clone this repo (so it knows who I am)
2. Add a commit: `git commit --allow-empty -m "This is an example commit"`
3. View the commit: `git log` ; You'll notice your name is with that commit
4. Amend the author of the commit: `git commit --amend --allow-empty --author="Awkanimus"`
5. View the commit: `git log` ; You'll notice the author is now me, but the "committer" is you.

This is often done to indicate that someone made the changes, and someone else is committing it for them.

**Review by AtlanteFr — APPROVED** [review]
